tests/sample_test/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+# create compile_commands.json for clang
+set(CMAKE_EXPORT_COMPILE_COMMANDS on)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(sample_test)
+
+# Make a helper variable for the app directory
+set(APP_DIR ../../app/main/src)
+
+# Add source files with test code
+file(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
+
+# Add module under the test
+
+# This can be done in several ways:
+
+# A. Add source files and header include directory via Cmake
+#
+# For example:
+#
+# target_include_directories(app PRIVATE ${APP_DIR}/some/path/header_dir)
+#
+# target_sources(app PRIVATE ${APP_DIR}/some/path/source_file.c)
+#
+# This is suitable if the module under test is a part of the application source
+# directory
+#
+# B. Use Kconfig symbol to select the module under test.
```

Comment:
For a slight improvement in readability, we could put a colon ":" instead of a comma after case letters. We could also add a hyphen in front and do a slight indent. Like this:
```
# - A: Add source files and header include directory via Cmake
#
#   For example:
#
#   target_include_directories(app PRIVATE ${APP_DIR}/some/path/header_dir)
#
#   target_sources(app PRIVATE ${APP_DIR}/some/path/source_file.c)
#
#   This is suitable if the module under test is a part of the application source
#   directory
#
# - B: Use Kconfig symbol to select the module under test.
```

Reply:
Thank your for suggestion, however my CMake formatter for some reason changes the dashes "-" into asterixes "*" in the comments and the result looks worse to me than the existing version. Also, any extra indent is also removed by the formatter.

I will leave it as it is.